addons/bft/functions/fnc_handleRegisteredModeChanged.sqf

```diff
@@ -42,7 +42,7 @@ if (_add) then {
         };
     } foreach GVAR(deviceData);
 } else { // if we remove one, we only have to check the already available devices
-    private ["_device","_i"];
+    private ["_device"];
```

Comment:
This private isn't needed. The _device variable doesn't need to live outside of the `for` scope.